README.md

```diff
@@ -63,8 +63,29 @@ ServerCommunity = 1
 IsBattleEyeEnabled = True
 ```
 
+### Ginfo Integration
+
+Ginfo Integration allows you to track the position of players on your server in real time on the [Ginfo Map](https://conanexiles.ginfo.gg)
+
+**Set up the Integration:**
+
+* You'll need to be the admin of a ginfo group
+* Open the panel of your group by clicking on your groups button in the left bar
+* Click on the "..." more button at the top right of the panel
+* Open the "Access Tokens" Menu
+* Create an access token
+* Copy your group's UID and access token to your **serverthrall.config** under the `ApiUploader` section
+
+```
+[ApiUploader]
+enabled = true
+ginfo_group_uid = <GROUP UID>
+ginfo_access_token = <ACCESS TOKEN>
+```
+
+Thats it! Restart ServerThrall and you should see markers appear in your group.
```

Comment:
I would remove this line. Right now server thrall actually constantly over-writes the config so you can't actually edit it while serverthrall is running anyone until that's fixed so telling them to restart server thrall kind of implies it could have already been running to begin with.